src/libraries/System.Transactions.Local/src/System/Transactions/DtcProxyShim/DTCInterfaces/IPrepareInfo.cs

```diff
@@ -0,0 +1,33 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Runtime.InteropServices;
+
+namespace System.Transactions.DtcProxyShim.DtcInterfaces;
```

Comment:
I don't have strong feelings either way - I tend to prefer using filed-scoped namespaces at least for new files, since there's no real downside to it (except maybe the inconsistency across files). I'll leave as-is for now, but if you prefer I revert, I'll do that.

Reply:
It is better to be consistent within a project at least. Otherwise, you are likely going to see external contributors submitting fixes to make things consistent.